### PR TITLE
Build ruby-bionic with ruby versions 2.7.2 and 3.0.0 - EN-44356

### PR DESCRIPTION
**Pull request by jsondheimer**
Added Dockerfiles to get ruby-bionic building with 2.7.2. and 3.0.0

### CODE REVIEW

File: ruby-bionic/2.7.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	\
+	&& cd / \
+	&& rm -r /usr/src/ruby \
+	\
+	&& gem update --system "$RUBYGEMS_VERSION" \
+	&& gem install bundler --version "$BUNDLER_VERSION" --force \
+	&& rm -r /root/.gem/
+
+# LABEL must be last for proper base image discoverability
+LABEL repository.socrata/ruby2.6.3=""
```
Comment (znep) [inline ruby-bionic/2.7.2/Dockerfile:103]: Don't forget to update the versions in the labels

**Comment by znep** [comment]
@jsondheimer asked for comments on rubygems / bundler versions:

 I'm not sure how much of that is a holdover from previous issues that required specific versions to work around bugs.  

I'd be inclined to probably just remove the rubygems update entirely and let it run with what comes with ruby.  Second choice would be update both without a version to get the latest, or explicitly to the latest 3.2.x

For bundler I'd be inclined to start off trying the latest 2.2.10 or just not specifying a version to always get the latest.  

My reasoning for preferring newer versions is we know nothing will use the images initially so it is a good chance to keep up to date, if folks upgrading images run into issues with those then we can decide how to proceed.  Otherwise we just end up with obsolete versions for longer which causes more problems in general as we keep running into issues solved in newer versions.

I'm on the fence about hard-coding the current latest vs just letting it float until/unless we run into an issue, in the end either would be fine.  I'm normally all for pinning versions of things but in this case that results in never being able to update downstream images without risking breaking things using them anyway, and the more out of date they get the bigger the breakage tends to be, so may as well take the lumps as they come.

**Comment by jsondheimer** [comment]
Bumped both to the latest versions of gem and bundler for now and corrected the labels. Will try to figure out if we even need to bother pinning gem and bundler versions per above.

**Comment by jsondheimer** [comment]
Considered pinning to latest minor version, but there doesn't appear to be a good way to do that for gem without a hacky workaround.

**Comment by mark-wagner** [comment]
I wonder why ruby-build isn't used but I guess this isn't the time to go down that rabit hole.

**Review by mark-wagner — APPROVED** [review]

**Review by dosieckityler — APPROVED** [review]
Giving my approval in the context that I see no red flags, but that I'm basically just now learning that we build ruby from raw sources in our images. :beginner:

**Pull request merged**